ghcide/src/Development/IDE/Types/Exports.hs

```diff
@@ -121,7 +136,8 @@ createExportsMapHieDb hiedb = do
         let mn = modInfoName $ hieModInfo m
             mText = pack $ moduleNameString mn
         fmap (wrap . unwrap mText) <$> getExportsForModule hiedb mn
-    return $ ExportsMap $ Map.fromListWith (<>) (concat idents)
+    let exportsMap = Map.fromListWith (<>) (concat idents)
+    return $ ExportsMap exportsMap $ buildModuleExportMap (concat idents)
```

Comment:
Similarly, this code could easily return a triple `(moduleName, identifierText, IdentInfo)` instead of deriving the module export map from the identifiers one

Reply:
addressed